libr/include/r_io.h

```diff
@@ -26,7 +26,7 @@
 #include <sys/ptrace.h>
 #endif
 
-#if defined(__APPLE__) || __sun || __FreeBSD__ || __OpenBSD__
+#ifndef __GLIBC__
```

Comment:
what about gnu/kfreebsd ?

Reply:
covered now by only using the enum on glibc on linux